index.rst

```diff
@@ -1,57 +1,343 @@
+As mentioned in the previous section, it is also used as a back-end for user-commanded actions performed using the `LOVE`_ interface.
+For example, when a user performs a telescope offset operation using LOVE, the actual action perform by LOVE system will the queueing of a :ref:`SAL Script <SAL_Scripts>` that then gets executed by the `ScriptQueue`_.
+The ScriptQueue is also the main tool observers will rely on to perform regular nighttime operations, such as tracking a target and obtaining a standardized set of observations.
+The `ScriptQueue`_ is flexible in how it handles and executes scripts but generally it runs sequentially.
+As soon as one script finishes, the next one begins until no scripts remain to be run.
+
 ..
-  Technote content.
+    This next bit feel repetitive..
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
+This architecture (or methodology) means the number functions that get executed by the scriptQueue will be very large and their complexity varies significantly.
+For instance, when operators need to prepare the observatory for calibrations or for night operation, they will add a rather complex script to the ScriptQueue that will perform all the required tasks (e.g. position telescope and dome, perform required fine tuning in positioning, open mirror covers and so on).
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+The Scheduler for the survey essentially adds scripts to the scriptQueue after each visit until told by the observer to stop.
 
-  Use the following syntax for sections:
+As mentioned already, interacting with the ScriptQueue is mainly done through :ref:`LOVE ScriptQueue user interface <fig-scriptqueueui>`.
+The interface allows users to stop, start and pause the ScriptQueue.
+It also displays the status of scripts and any errors that occur.
+Shuffling the order of queued scripts is also possible.
+Lastly, users can quickly relaunch previously run scripts without having to re-enter any modifications to the default configuration.
 
-  Sections
-  ========
+.. figure:: /_static/ScriptQueueUI.png
+   :name: fig-scriptqueueui
+   :target: ../_images/ScriptQueueUI.png
+   :alt: LOVE ScriptQueue user interface
 
-  and
+   A screenshot of the LOVE interface to interact with the ScriptQueue.
 
-  Subsections
-  -----------
+In general, scripts require minimum interaction to be executed except, of course, for the occasional configuration.
+A database of configurations will be available for users to execute the most common set of operations.
+Users also have the possibility to edit configurations from the user interface, which also provides on-the-fly schema validation.
 
-  and
+In general, using the ScriptQueue requires some familiarity with the observatory system and minimal set of software development skills.
```

Comment:
The previous paragraph also started with "In general" maybe start it differently? Maybe "Furthermore, ?"

Reply:
rephrased the above line so this "In general" can stay.